definitions/procedures/puppet/remove_puppet_data.rb

```diff
@@ -0,0 +1,21 @@
+module Procedures::Puppet
+  class RemovePuppetData < ForemanMaintain::Procedure
+    metadata do
+      description 'Remove Puppet data'
+    end
+
+    def run
+      execute!('foreman-rake purge:puppet')
+      execute!('rm -r ' + files_to_purge.join(' '))
+    end
+
+    private
+
+    def files_to_purge
+      %w[
+        /etc/puppetlabs
```

Comment:
My biggest issue with this is that files which belong to puppet-agent are also removed. For now this is probably OK, but I wonder what the output of `rpm -qV puppet-agent` is after running the scenario.